tests/ui/async-await/issues/issue-67611-static-mut-refs.rs

```diff
@@ -9,6 +9,7 @@ fn is_send_sync<T: Send + Sync>(_: T) {}
 
 async fn fun() {
     let u = unsafe { A[async { 1 }.await] };
+    //~^ WARN creating a reference to mutable static is discouraged [static_mut_refs]
```

Comment:
Since this test is unrelated, let's fix (or `allow(..)` and mark `FIXME`) this and other similar uses instead.